lib/web_ui/skwasm/helpers.h

```diff
@@ -49,7 +49,7 @@ inline SkSamplingOptions samplingOptionsForQuality(FilterQuality quality) {
     case FilterQuality::none:
       return SkSamplingOptions(SkFilterMode::kNearest, SkMipmapMode::kNone);
     case FilterQuality::low:
-      return SkSamplingOptions(SkFilterMode::kNearest, SkMipmapMode::kNearest);
+      return SkSamplingOptions(SkFilterMode::kLinear, SkMipmapMode::kNone);
```

Comment:
You definitely want nearest for mipmap mode low. The framework draws all images with filter quality low and mipmapping is required for correct rendering if the images are shrunk from their original size substantially.

Reply:
Although looking at the skia options, maybe we're handling this wrong:

enum SkLegacyFQ {
    kNone_SkLegacyFQ   = 0,    //!< nearest-neighbor; fastest but lowest quality
    kLow_SkLegacyFQ    = 1,    //!< bilerp
    kMedium_SkLegacyFQ = 2,    //!< bilerp + mipmaps; good for down-scaling
    kHigh_SkLegacyFQ   = 3,    //!< bicubic resampling; slowest but good quality

    kLast_SkLegacyFQ = kHigh_SkLegacyFQ,
};

Ignore me!